src/Microsoft.OData.Edm/Csdl/CsdlJsonWriter.cs

```diff
@@ -49,13 +50,30 @@ protected override void WriteCsdl()
             WriteReferenceElements();
 
             // It also MAY contain members for schemas.
-            WriteSchemata();

```

Comment:
Schemata is correct plural of Schema, right?

Reply:
@xuzhg That is correct. However, I have used `WriteSchemas` and `WriteSchemasAsync` to be the same way they are named in [CsdlXmlWriter](https://github.com/OData/odata.net/blob/fix/2684-csdlwriter-async-apis/src/Microsoft.OData.Edm/Csdl/CsdlXmlWriter.cs#L19) where we have [WriteSchemas](https://github.com/OData/odata.net/blob/fix/2684-csdlwriter-async-apis/src/Microsoft.OData.Edm/Csdl/CsdlXmlWriter.cs#L246) and [WriteSchemasAsync](https://github.com/OData/odata.net/blob/fix/2684-csdlwriter-async-apis/src/Microsoft.OData.Edm/Csdl/CsdlXmlWriter.cs#L259)